components/src/core/hero/hero.component.scss

```diff
@@ -1,61 +1,59 @@
-@import '~@pxblue/colors/palette';
-
 :host {
     flex: 1 1 0px;
 }
 
-.pxb-root {
+.pxb-hero {
     display: flex;
     flex-direction: column;
     align-items: center;
     justify-content: flex-start;
     font-size: 1rem;
     overflow: hidden;
-    color: map-get($pxb-gray, 500);
     padding: 16px 8px;
-}
-
-.pxb-primary-wrapper {
-    padding: 3px;
-    $normal: 32px;
-    $large: 72px;
-    color: map-get($pxb-gray, 800);
-    border-radius: 50%;
-    margin-bottom: 5px;
-    text-align: center;
-}
-
-.pxb-channel-value-wrapper {
-    display: flex;
-    color: map-get($pxb-gray, 800);
-}
 
-// Label Styles
-.pxb-label {
-    font-size: inherit;
-    line-height: 1.2;
-    letter-spacing: 0;
-    font-weight: 600;
-    width: 100%;
-    text-align: center;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    margin: 0;
-}
+    .pxb-hero-primary-wrapper {
+        padding: 3px;
```

Comment:
4px to get us on the material spacing units?

Reply:
Good idea, updated to 4